Maple2.Server.Game/Packets/CubePacket.cs

```diff
@@ -486,6 +490,26 @@ public static ByteWriter UpdateHomeAreaAndHeight(byte area, byte height) {
         return pWriter;
     }
 
+    public static ByteWriter CreateBlueprint(long itemUid, ItemBlueprint blueprint) {
+        var pWriter = Packet.Of(SendOp.ResponseCube);
+        pWriter.Write<Command>(Command.CreateBlueprint);
+        pWriter.WriteByte(1);
```

Comment:
would it be safe to assume it's UgcMapError too?

Reply:
i guess so but error 1 is s_empty_string, that's ok?

Reply:
yeah I would think. no error for creating blueprint checks out